rust/main/agents/relayer/src/msg/op_queue.rs

```diff
@@ -74,27 +75,55 @@ impl OpQueue {
         // The other consideration is whether to put the channel receiver in the OpQueue or in a dedicated task
         // that also holds an Arc to the Mutex. For simplicity, we'll put it in the OpQueue for now.
         let mut message_retry_requests = vec![];
+
         while let Ok(message_id) = self.retry_rx.lock().await.try_recv() {
```

Comment:
nit: this should've been renamed before but can you rename `message_id` to something more accurate, as it's actually a `MessageRetryRequest`